feat(serverless): add serverless job retries MTA-6904 (#6602)
scaleway/docs-content: pages/serverless-jobs/how-to/create-job.mdx

     1. Add [commands and arguments](/serverless-jobs/concepts/#commands-and-arguments) to your job. They will be executed every time your job is run.
     2. Set a **maximum duration** to your job to stop it automatically if it does not complete within this limit.
     </TabsTab>
     <TabsTab label="Retries">

>      - Set the number of times you want your job to be rerun in case of failure. The default value is 0, and the maximum value is 5.

Reviewer: ```suggestion
     - Set the number of times the job retries on failure. The default value is 0, and the maximum value is 5.
```

    1. Add a **startup command** to your job. It will be executed every time your job is run.
    2. Set a **maximum duration** to your job to stop it automatically if it does not complete within this limit.
    </TabsTab>
    <TabsTab label="Retries">

>      - Set the number of times you want your job to be rerun in case of failure. The default value is 0, and the maximum value is 5.

Reviewer: ```suggestion
     - Set the number of times the job retries on failure. The default value is 0, and the maximum value is 5.
```
